samples/matter/light_bulb/README.rst

```diff
@@ -73,33 +73,43 @@ Configuration
 
 |config|
 
+
+.. _matter_light_bulb_build_types:
+
 Matter light bulb build types
 =============================
 
 .. matter_light_bulb_sample_configuration_file_types_start
 
-The sample uses different configuration files depending on the supported features.
-Configuration files are provided for different build types and they are located in the application root directory.
-
-The :file:`prj.conf` file represents a ``debug`` build type.
-Other build types are covered by dedicated files with the build type added as a suffix to the ``prj`` part, as per the following list.
-For example, the ``release`` build type file name is :file:`prj_release.conf`.
-If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
-
-.. include:: /config_and_build/modifying.rst
-   :start-after: build_types_overview_start
-   :end-before: build_types_overview_end
-
-Before you start testing the application, you can select one of the build types supported by the sample.
-This sample supports the following build types, depending on the selected board:
-
-* ``debug`` -- Debug version of the application - can be used to enable additional features for verifying the application behavior, such as logs or command-line shell.
-* ``release`` -- Release version of the application - can be used to enable only the necessary application functionalities to optimize its performance.
-* ``no_dfu`` -- Debug version of the application without Device Firmware Upgrade feature support - can be used for the nRF52840 DK, nRF5340 DK, nRF7002 DK, and nRF21540 DK.
-
-.. note::
-    `Selecting a build type`_ is optional.
-    The ``debug`` build type is used by default if no build type is explicitly selected.
+The sample does not use a single :file:`prj.conf` file.
+Configuration files are provided for different build types, and they are located in the sample root directory.
+Before you start testing the application, you can select one of the build types supported by the application, depending on the building method.
```

Comment:
```suggestion
Before you start testing the application, you can select one of the build types supported by the application.
```
Having the "depending on the building method" here makes it sound like the building method affects which build types are supported.

Maybe we should have the default mentioned here as well? Or in the Selecting a build type section?

Should the starting five lines of this section be a separate include file? Makes it easier to maintain in the long run, especially if additional samples get build types.

Reply:
Edit applied.
I removed the default line from the table and placed "default" next to the build type name in the table.
I'm unsure about making this a separate include file, as these lines are different for apps and for samples, and these lines are already included through an include in light switch and other samples. I don't know if include within include would really work...